compal/__init__.py

```diff
@@ -758,16 +958,16 @@ def set_ipv4_dhcp(self, addr_start, addr_end, num_devices, lease_time, enabled):
         )
 
     def set_ipv6_dhcp(
-        self,
-        autoconf_type,
-        addr_start,
-        addr_end,
-        num_addrs,
-        vlifetime,
-        ra_lifetime,
-        ra_interval,
-        radvd,
-        dhcpv6,
+            self,
```

Comment:
Sorry for this kinds of errors. I thought i rechanged them correctly. Perhaps pycharm wanted more spaces... hummm